data/controller.xql

```diff
@@ -40,8 +40,7 @@ if (ends-with($exist:resource, ".xml")) then
                 <param name="exist:stop-on-warn" value="no"/>
                 <param name="exist:stop-on-error" value="no"/>
 </parameters>, <attributes></attributes>, "method=xml media-type=application/xml"),
-        $saved := xmldb:store("/db"||$exist:prefix||$exist:controller||string-join(tokenize($exist:path, '/')[position() != last()], '/'), $exist:resource, $filtered)
-    return doc($saved)
```

Comment:
This is most probably an error.

Reply:
Yes, thanks for spotting. Re-added it in  815a14c